gems/pending/metadata/linux/OpenstackConfigurationParser.rb

```diff
@@ -0,0 +1,200 @@
+module MiqLinux
+  class OpenstackConfigurationParser
```

Comment:
I don't think this file belongs here. The metadata/linux directory is intended to be linux guest OS specific. This file is neither Linux nor guest OS specific - it's actually OpenStack specific. As such it probably should reside under models/ems_refresh/parsers - in an Openstack infra specific file.